lib/instrumentation/generic-span.js

```diff
@@ -14,7 +13,10 @@ function GenericSpan (agent, ...args) {
     : {}
 
   this._timer = new Timer(opts.timer, opts.startTime)
-  this._context = TraceParent.startOrResume(opts.childOf, agent._conf) // _context is used by the OT bridge, and should unfortunately therefore be considered public
+
+  // _context is used by the OT bridge, and should unfortunately therefore be considered public
+  this._context = TraceContext.startOrResume(opts.childOf, agent._conf, opts.tracestate)
+
```

Comment:
Here's where the old TraceParent object was added to a `Transaction` or `Span`  object.  We changed this to use a `TraceContext` object instead. `TraceContext` behaves the same as `TraceParent`, but all accepts a `tracestate` header to parse.